sw/cheri/sim_boot_stub/boot.S

```diff
@@ -5,7 +5,7 @@
 	.p2align 2
     .type start,@function
 start:
-	auipcc           cra, 0
-	li               t0, 0x00101000
-	csetaddr         cra, cra, t0
-	cjr              cra
+  auipcc           ct1, 0
```

Comment:
Currently we have tab indentation for all .S files. We should decide on a style and reformat all .S files.

Reply:
We should follow the lowRISC style guide as we do for all other lowRISC repositories, so space indentation. Yes we do need to reformat this repository.

I think we may make an exception for sonata-software as it's more important for it be compatible with CHERIoT RTOS upstream which uses tabs.